plugins/Diagnostics/Diagnostic/RecommendedFunctionsCheck.php

```diff
@@ -72,6 +72,10 @@ private function getHelpMessage($function)
             'gzopen'         => 'Installation_SystemCheckZlibHelp',
         );
 
-        return $this->translator->translate($messages[$function]);
+        $translation_params = array(
+            'shell_exec'     => ["<a href='https://matomo.org/faq/troubleshooting/how-to-make-the-diagnostic-managing-processes-via-cli-to-display-ok/'>", "</a>"]
```

Comment:
I guess this one should have a `target='_blank'`, otherwise clicking it will leave Matomo

Reply:
And noopener, right?

Reply:
Thanks 👍

Reply:
Understanding from https://github.com/matomo-org/matomo/pull/18218 we don't need noreferrer anymore.

Reply:
Ah, indeed (at least as long as people use modern browsers)

Reply:
I saw your other comment from email @Findus23 but I guess maybe you removed it later? The translation text isn't currently used anywhere other than for `shell_exec` from my usages search but I could leave that translation there and create another if you're aware of any plugins or other places that may use it. I think possibly better to not leave unused translations in there but I'm open to other ideas.

Reply:
I noticed too afterwards that it is only used for shell_exec, that's why I deleted it. Maybe the translation key could be changed to reflect this (as we are already invalidating it). But leaving everything as-is also seems fine by me.

Reply:
Good idea.